test/OpenTelemetry.ResourceDetectors.Azure.Tests/AzureResourceDetectorTests.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="AzureResourceDetectorTests.cs" company="OpenTelemetry Authors">
```

Comment:
Should we consider naming as AppServiceDetectorTests?

Reply:
I was not anticipating lot of tests for each detector so, was just planning to include in single file. I can separate it out.

Reply:
discussed offline. will keep it as it is for now. may separate it out later